### DISCUSSION TITLE
How to solve this situation?

### DISCUSSION THREAD

**Discussion by iamxiaojianzheng** [LazyVim/LazyVim#3513] [Q&A] [answered]
I hope to be able to use the normal `vs` function，Split a window that displays half the width.

Why this effect occurs and how I should turn it off.

![image](https://github.com/LazyVim/LazyVim/assets/29248764/ea3822ac-d4b6-4e0b-86e9-36a400968dc6)

**Comment by folke** [comment]
You pressed `<leader>m` to toggle mazimize

**Reply by iamxiaojianzheng** [reply]
> You pressed `<leader>m` to toggle mazimize您按 `<leader>m` 下切换 mazimize

I tried, but it didn't work out. 

And when I manually adjust the width by 'Ctrl+Left/Right' and then use 'Ctrl+h/l' to switch the window, it is back to the way it was before.

**Reply by dpetka2001** [reply]
Just delete the directory I mentioned in my other comment and your problem should go away.

**Reply by iamxiaojianzheng** [reply]
> Just delete the directory I mentioned in my other comment and your problem should go away.只需删除我在其他评论中提到的目录，您的问题就会消失。

I've already wiped out the files in this folder.

**Reply by dpetka2001** [reply]
Did you update LazyVim to the latest release? If you did, this should not happen any more.

**Reply by iamxiaojianzheng** [reply]
> Did you update LazyVim to the latest release? If you did, this should not happen any more.您是否将 LazyVim 更新到最新版本？如果你这样做了，这不应该再发生了。

After exiting vim, just empty the folder again.

**Comment by dpetka2001** [comment, marked as answer]
You probably have used the `maximize` function before the recent LazyVim update that provides a solution for this and now the session manager remembers that. Delete your sessions in `~/.local/state/nvim/sessions/` directory and let Neovim re-create them. If you have updated to latest LazyVim release, this should not happen again.

**Reply by iamxiaojianzheng** [reply]
> You probably have used the `maximize` function before the recent LazyVim update that provides a solution for this and now the session manager remembers that. Delete your sessions in `~/.local/state/nvim/sessions/` directory and let Neovim re-create them. If you have updated to latest LazyVim release, this should not happen again.在最近的 LazyVim 更新之前，您可能已经使用过该 `maximize` 函数，该函数为此提供了解决方案，现在会话管理器记住了这一点。删除目录中的 `~/.local/state/nvim/sessions/` 会话，让 Neovim 重新创建它们。如果您已更新到最新的 LazyVim 版本，则不应再次发生这种情况。

I am using Windows, does it correspond to this path? 
`.\ AppData\Local\nvim-data\sessions`

**Reply by dpetka2001** [reply]
No idea what the path is on Windows. Do `:echo stdpath("state")` from inside Neovim and whatever is printed out, it should be the directory where the `sessions` directory resides.

**Reply by iamxiaojianzheng** [reply]
> No idea what the path is on Windows. Do `:echo stdpath("state")` from inside Neovim and whatever is printed out, it should be the directory where the `sessions` directory resides.不知道 Windows 上的路径是什么。从 Neovim 内部执行 `:echo stdpath("state")` ，无论打印出什么，它都应该是 `sessions` 目录所在的目录。

That's right. That's the path.

I also found that this does not happen when I edit lua files. 

The function of 'vs' is the normal display effect. 

But when I edit the vue file, it will look like the picture.

**Reply by dpetka2001** [reply]
Delete all the files in that directory.

**Reply by iamxiaojianzheng** [reply]
> Delete all the files in that directory.删除该目录中的所有文件。

Thank you for your reply. It has been successfully solved.
